helmfile.d/stacks/monitoring-prometheus.yaml

```diff
@@ -14,6 +14,9 @@ templates:
     needs:
       - monitoring/networkpolicy
       - monitoring/podsecuritypolicy
+      {{- if .Values | get "kubeStateMetrics.clusterAPIMetrics.enabled" false }}
```

Comment:
Why is this only necessary for capi? Is it installed by kubespray? If so I'd say we can add it as a dependency even there's a non capi cluster.

Reply:
This chart only includes metrics for cluster-api environments https://github.com/elastisys/compliantkubernetes-apps/tree/main/helmfile.d/charts/kube-state-metrics-extra-resource-metrics